### ISSUE TITLE
Download links order

### ISSUE THREAD

**Issue by graphixx-mgn** [WWBN/AVideo#4363]
Today I have found a weird order of downloading links. Once encoder finished its work two different videos appeared at MyVideo page but their download links (.mp4) have different order. I checked table videos of streamer database but did not found any links there. So, maybe these links are being generated based on filenames of the videos in the "video" directory. 

**To Reproduce**
Encode 2 videos by encoder using "Dynamic MP4" profile. It generates several files of a video with different resolutions.
1. Go to 'MyVideo' page and check order of download buttons.
2. Then can go to the video page then click on "Download" button. The order here is same as in MyVideo but weird as well.

I'm not sure is it a bug or not. 

**Expected behavior**
In my opinion it does not matter what order is (ASC or DESC) but I expected to see the same links order of the different videos.

**Screenshots**
![image](https://user-images.githubusercontent.com/30086544/104664227-8517c780-56f0-11eb-8a20-625016e9cee0.png)

**Comment by graphixx-mgn** [comment]
Probably the cause is in function _glob of file ./objects/functions.php. It  uses "readdir" which seems does not preserve an order.

**Comment by graphixx-mgn** [comment]
Used rsort($array, SORT_NATURAL); after getting filenames solved my issue.

Now the links are ordered by resolution DESC. 
Also there was another disadvantage of the issue is a different default resolution of played video. For example in relation to my screenshot above:
* First video is being played in 240p
* Second video is being played in 1080p

So, now the problem disappeared as well.

**Comment by DanielnetoDotCom** [comment]
Try this update

**Comment by graphixx-mgn** [comment]
Thanks for your reply. 
Just checked the update... Now the links on the MyVideo page are sorted.
![16-01-2021 2-00-28](https://user-images.githubusercontent.com/30086544/104778115-a9ca7880-579e-11eb-9c37-6b09f092ca35.png)

But as I wrote today I've found another affection of the issue. 
![16-01-2021 1-57-57](https://user-images.githubusercontent.com/30086544/104778051-8acbe680-579e-11eb-9d53-2a325b892846.png)
As far as first resolution is 240p a video player uses the resolution by default when starts playing video. 
I'd prefer to be the highest resolution by default but understand it's up to situation/bandwidth etc.

I've seen there is a plugin to switch resolution of playing video so it could help.

Respectfully yours.
